common/api/core-bentley.api.md

```diff
@@ -1178,6 +1178,13 @@ export namespace JsonUtils {
 // @public
 export type Listener = (...arg: any[]) => void;
 
+// @beta
+export type ListenerType<TEvent extends {
```

Comment:
Why make generic instead of using `BeEvent` explicitly? (`BeUiEvent` derives from `BeEvent`.)

Reply:
It's to support narrower event types, like https://github.com/iTwin/presentation/blob/b9598f6702bea0f466abae02378a0c958f8e5fe9/packages/shared/src/shared/Event.ts#L10
Take a look at test named `Narrowed event type`